src/LogMonitor.js

```diff
@@ -122,17 +137,7 @@ export default class LogMonitor extends Component {
     }
   }
 
-  componentDidUpdate() {
-    const node = this.refs.container;
-    if (!node) {
-      return;
-    }
-    if (this.scrollDown) {
-      const { offsetHeight, scrollHeight } = node;
-      node.scrollTop = scrollHeight - offsetHeight;
-      this.scrollDown = false;
-    }
-  }
+  componentDidUpdate = this.scroll;
```

Comment:
Yes, sorry, I wasn't clear enough. I meant that one disadvantage of `componentDidUpdate = this.scroll` (as I did before) is that it creates an impression that `this.scroll` is called as `this.scroll(prevProps, prevState)` as `componentDidUpdate` passes those arguments.